edx_proctoring/templates/emails/proctoring_attempt_status_email.html

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}
+
+Dear {{ student_name }},
+
+    This email is to notify that status of {{ exam_name }} for {{ course_name }} has changed to {{ status }} on {{ date }}.
+
+Thank you.
+
+
```

Comment:
@catong sorry to be vague, we're just using placeholder content for now. I pinged @griffresch on this and he'll try to propose some initial content for review.